core-backend/src/funcs.rs

```diff
@@ -65,7 +65,7 @@ pub(crate) fn debug<E: Ext>(ext: LaterExt<E>) -> impl Fn(i32, i32) -> Result<(),
             let mut data = vec![0u8; str_len];
             ext.get_mem(str_ptr, &mut data);
             let debug_str = unsafe { String::from_utf8_unchecked(data) };
-            log::debug!("DEBUG: {}", debug_str);
+            log::debug!(target: "wasm", "DEBUG: {}", debug_str);
```

Comment:
I'd better keep it without custom target

for rust_log, it's just name of the crate for a filter

`export RUST_LOG=gear_core_backend=debug`

Reply:
My idea was to distinguish gr_debug messages among a ton of others and always show them in gtest.
But I'll revert it back, ok.

Reply:
Ah ok, you can name it gwasm-debug or smth to make it unique

Reply:
Done with `gwasm_debug`.